.changeset/lovely-cameras-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@finos/legend-studio-extension-query-builder": minor
```

Comment:
Lets add `"@finos/legend-query": minor` here and delete the file `stupid-geese-travel`